recipes/bookstore/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "bookstore" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac8d72754a283ccccc6483d518d952bfdc5ba072a1fff2fbe736fa7e02c69d9c
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - future
+    - ipython>=5.0
```

Please leave a space between name and spec
```suggestion
    - ipython >=5.0
```